sound/soc/codecs/rt712-sdca.h

```diff
@@ -207,6 +238,11 @@ enum rt712_sdca_hw_id {
 
 #define RT712_PART_ID_713 0x713
 
+enum rt712_sdca_version {
+	RT712_VA,
+	RT712_VB,
+};
+
```

Comment:
is there anything that differentiates the VA and VB at the device_ID level?
Also is there a way to know if the dmic is supported or not based on ACPI information?
not sure how we would change the machine driver to add the dmic path or not?

@bardliao what do you think?

Reply:
We could use `dai_info->quirk` to make the DMIC be optional, like https://github.com/thesofproject/linux/pull/4848/commits/47bae326da237ca6cf0414dc5a8a823d1ff83a39

Reply:
yes but that quirk is defined at the machine driver level. It's too late to select the topology, which is entirely based on ACPI information.